src/main/java/net/sf/jabref/importer/UnlinkedFilesCrawler.java

```diff
@@ -60,12 +61,24 @@ public CheckableTreeNode searchDirectory(File directory, UnlinkedPDFFileFilter f
             return null;
         }
 
-        List<File> files = Arrays.asList(directory.listFiles(ff));
+        File[] filesArray = directory.listFiles(ff);
```

Comment:
What is wrong with the the previous code? I think there is no need to convert the array to a list since we just iterate over it anyway.